js/adopt-a-dog-blog.js

```diff
@@ -0,0 +1,171 @@
+const fields = [
+    {
+        name: 'title',
+        label: 'Title'
+    },
+    {
+        name: 'image',
+        label: 'Image Link'
+    },
+    {
+        name: 'text',
+        label: 'Blog Text'
+    }
+];
+
+const form = document.createElement('form');
+
+document.body.appendChild(form);
```

Comment:
The above was not required for this assignment, but I will not take off points for it.